### PR TITLE
Added click handlers on frontend page

### PR DESCRIPTION
**Pull request by NikitaDenysenko**

### CODE REVIEW

File: src/pages/home/categories-list/category-item/category-item.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ const CategoryItem = ({
   language
 }) => {
   const [image, setImage] = useState(categoryImageUrl);
+  const history = useHistory();
+
+  const path = `/${categoryUrl}`;
+
+  const routeChange = () => {
```
Comment (vasylshpak) [inline src/pages/home/categories-list/category-item/category-item.js:19]: maybe rename this to 
```suggestion
  const changeRoute = () => {
```
it's sounds better to me.

Comment (vasylshpak) [reply src/pages/home/categories-list/category-item/category-item.js:19]: or
```suggestion
  const routeChangeHandler = () => {
```

Comment (vasylshpak) [reply src/pages/home/categories-list/category-item/category-item.js:19]: or
```suggestion
  const changeRouteHandler= () => {
```

**Comment by vasylshpak** [comment]
please resolve conversations that are closed by pressing "Resolve conversation" button

File: src/components/model-item/model-item.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ const ModelItem = ({ model }) => {
 
   const dispatch = useDispatch();
   const styles = useStyles();
+  const history = useHistory();
+  const path = `/${model.category.name[1].value.toLowerCase()}/${model.name[1].value.toLowerCase()}`;
 
   const handleClick = (selectedModel) => {
```
[the remark this answers is not in the document]
Comment (NikitaDenysenko) [reply src/components/model-item/model-item.js]: done

**Review by Iryna-Bzdyr — APPROVED** [review]

**Review by vasylshpak — APPROVED** [review]

**Review by nadiyafomenko — APPROVED** [review]

**Pull request merged**